src/zmq/zmqnotificationinterface.cpp

```diff
@@ -110,6 +116,12 @@ bool CZMQNotificationInterface::Initialize()
 void CZMQNotificationInterface::Shutdown()
 {
     LogPrint(BCLog::ZMQ, "zmq: Shutdown notification interface\n");
+
+#ifdef ENABLE_WALLET
+    if (pwalletMain)
```

Comment:
nit: missing braces `{` and `}` for the if block, also this need adjustment for multiwallet.